fed/proxy/barriers.py

```diff
@@ -199,69 +159,18 @@ async def send(
             f'Sending {send_log_msg} with{"out" if not self._tls_config else ""}'
             ' credentials.'
         )
-        dest_addr = self._cluster[dest_party]['address']
-        dest_party_grpc_config = self.setup_grpc_config(dest_party)
         try:
-            tls_enabled = fed_utils.tls_enabled(self._tls_config)
-            grpc_options = dest_party_grpc_config['grpc_options']
-            grpc_options = get_grpc_options(retry_policy=self.retry_policy) if \
-                grpc_options is None else fed_utils.dict2tuple(grpc_options)
-
-            if dest_party not in self._stubs:
-                if tls_enabled:
-                    ca_cert, private_key, cert_chain = fed_utils.load_cert_config(
-                        self._tls_config)
-                    credentials = grpc.ssl_channel_credentials(
-                        certificate_chain=cert_chain,
-                        private_key=private_key,
-                        root_certificates=ca_cert,
-                    )
-                    channel = grpc.aio.secure_channel(
-                        dest_addr, credentials, options=grpc_options)
-                else:
-                    channel = grpc.aio.insecure_channel(dest_addr, options=grpc_options)
-                stub = fed_pb2_grpc.GrpcServiceStub(channel)
-                self._stubs[dest_party] = stub
-
-            response = await send_data_grpc(
-                data=data,
-                stub=self._stubs[dest_party],
-                upstream_seq_id=upstream_seq_id,
-                downstream_seq_id=downstream_seq_id,
-                metadata=dest_party_grpc_config['grpc_metadata'],
-            )
+            response = await self.proxy_instance.send(
+                dest_party, data, upstream_seq_id, downstream_seq_id)
         except Exception as e:
             logger.error(f'Failed to {send_log_msg}, error: {e}')
             return False
         logger.debug(f"Succeeded to send {send_log_msg}. Response is {response}")
```

Comment:
`Succeeded to send {send_log_msg}.` -> `Succeeded to {send_log_msg}.`: redundant `send`

Reply:
Let's fix the legacy typos later in another PR ~